.github/workflows/publish-release-image-wrapper.yaml

```diff
@@ -1,12 +1,32 @@
 name: "Create and Publish Release Docker Image"
 
 on:
-  workflow_call:
   workflow_dispatch:
-
 jobs:
-  to_be_filled_out:
-    steps:
-      - name: This workflow will be filled out in https://github.com/tenstorrent/tt-metal/pull/15013
-        run: |
-          echo "NOOP"
+  static-checks:
+    uses: ./.github/workflows/all-static-checks.yaml
+    secrets: inherit
+  build-artifact:
+    needs: static-checks
+    uses: ./.github/workflows/build-artifact.yaml
+    secrets: inherit
+  build-wheels:
```

Comment:
Then why not a separate branch to compile the wheels for 22.04, or do we have a separate to do for that?

Reply:
I was under the assumption that 22.04 is not yet supported therefore avoided that and limited `build-wheels` to `os: [ubuntu-20.04]` Do we support 22.04 and ready for releasing it?

Or did you mean something else?

Reply:
Because this particular E2E should work in 22.04. I don't mind doing it later, but there should be a todo so we don't drop the ball

Reply:
https://github.com/tenstorrent/tt-metal/issues/15708